runtime/optwin.vim

```diff
@@ -1254,6 +1254,8 @@ call <SID>AddOption("isfname", gettext("specifies the characters in a file name"
 call <SID>OptionG("isf", &isf)
 call <SID>AddOption("isident", gettext("specifies the characters in an identifier"))
 call <SID>OptionG("isi", &isi)
+call <SID>AddOption("isexpand", gettext(gettext("specifies the characters used for text completion expansion"))
```

Comment:
`gettext()` called twice. And how is "text completion expansion" less ambiguous?

Reply:
oops and any suggestions for a better expression 🤔

Reply:
How about just mentioning `complete_match()` here?